src/FakeItEasy/FakeItEasy.csproj

```diff
@@ -280,7 +280,10 @@
   <ItemGroup />
   <ItemGroup>
     <EmbeddedResource Include="Creation\CastleDynamicProxy\DynamicProxyResources.resx" />
-    <EmbeddedResource Include="ExceptionMessages.resx" />
+    <EmbeddedResource Include="ExceptionMessages.resx">
+      <Generator>PublicResXFileCodeGenerator</Generator>
+      <LastGenOutput>ExceptionMessages1.Designer.cs</LastGenOutput>
+    </EmbeddedResource>

```

Comment:
> The .Designer.cs file is actually unlinked from the .resx in version control

They seem to be unlinked indeed, but it has nothing to do with version control. For now, you can just do as @blairconrad suggested. We'll fix the resx/designer.cs link later (not sure how it got unlinked)